server/drivers/hd44780.c

```diff
@@ -112,6 +112,52 @@ unsigned char HD44780_scankeypad(PrivateData *p);
 static int parse_span_list(int *spanListArray[], int *spLsize, int *dispOffsets[], int *dOffsize, int *dispSizeArray[], const char *spanlist);
 
 
+static const struct ModelMapping {
+	const char *name;
+	int model;
+} model_mapping[] = {
+	{ "default",      HD44780_MODEL_DEFAULT },
+	{ "standard",     HD44780_MODEL_DEFAULT },
+
+	{ "extended",     HD44780_MODEL_EXTENDED },
+	{ "ks0073",       HD44780_MODEL_EXTENDED },
+
+	{ "winstar_oled", HD44780_MODEL_WINSTAR_OLED },
+	{ "weh00xxyya",   HD44780_MODEL_WINSTAR_OLED },
+	{ "winstar",      HD44780_MODEL_WINSTAR_OLED },
+	{ "winstaroled",  HD44780_MODEL_WINSTAR_OLED },
+
+	{ "pt6314_vfd",   HD44780_MODEL_PT6314_VFD },
+	{ "pt6314",       HD44780_MODEL_PT6314_VFD },
+	{ "pt6314vfd",    HD44780_MODEL_PT6314_VFD },
+
```

Comment:
Should we really have that many synonyms for winstar and pt6314? While it might be convenient, I think it can also cause confusion whether configurations are equivalent or not. At least we shouldn't have synonyms that aren't mentioned in the documentation. I recommend dropping them here.